MAINTAINERS.md

```diff
@@ -1,19 +1,18 @@
-This file lists how the chef-server project is maintained. When making changes to the system, this file tells you who needs to review your patch - you need a simple majority of maintainers for the relevant subsystems to provide a :+1: on your pull request. Additionally, you need to not receive a veto from a Lieutenant or the Project Lead.
+This file lists how the chef-server project is maintained. When making changes to the system, this file tells you who needs to review your patch - you need a simple majority of maintainers for the relevant subsystems to provide a :+1: on your pull request.
```

Comment:
I think we should just say "2 👍".  A majority means 4 of those on the maintainer list need to +1 which means basically nothing would get merged these days.